src/components/layouts/collection-page-layout.tsx

```diff
@@ -222,6 +222,10 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
     tags: railsTags = [],
   } = course
 
+  const communityResource = ['free', 'community-resource'].includes(
```

Comment:
there is no "community resource" state. it's either free or pro, that's a label for free.

Reply:
"community resource" is marketing not database